data_collection/gazette/spiders/pa_cameta.py

```diff
@@ -0,0 +1,80 @@
+import datetime
+import re
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+RE_EDITION_NUMBER = re.compile(r"Edi..o\s*(\d+)")
+RE_EXT_DATE = re.compile(r"\s+(\d+)\sde\s(\w+)\sde\s(\d+)")
+
+
+class PaCameta(BaseGazetteSpider):
+    TERRITORY_ID = "1502103"
+    name = "pa_cameta"
+    allowed_domains = ["prefeituradecameta.pa.gov.br"]
+    start_date = datetime.date(2023, 1, 12)
+
+    BASE_URL = "https://prefeituradecameta.pa.gov.br/diario-oficial-do-municipio/"
+
+    def start_requests(self):
+        url = f"{self.BASE_URL}"
+        yield scrapy.Request(url, callback=self.parse)
+
+    def parse(self, response):
+        a_elements = response.css("#post-20884 > div > ul > li > a")
+        for a in a_elements:
+            text = a.css("a::text").get()
+            href = a.css("a::attr(href)").get()
+            # This link seems to be a test. will be ignored
+            if "12 de abril de 2022" in text:
+                continue
+            gazette_edition_number = PaCameta.get_edition_number(text)
+            gazette_date = PaCameta.get_date(text)
+            gazette_url = href
+            if self.start_date <= gazette_date <= self.end_date:
+                yield Gazette(
+                    date=gazette_date,
+                    edition_number=gazette_edition_number,
+                    file_urls=[gazette_url],
+                    is_extra_edition=False,
+                    power="executive",
```

Comment:
```suggestion
                    power="executive_legislative",
```
[modificação] Nos diários, tem um bloco escrito "Atos do Executivo" deixando sugestivo que no mesmo documento podem ter também atos do legislativo. Quando for assim, poderia já deixar como `executive_legislative` por antecipação, mesmo que não encontre casos. 

Mas encontrei um dando uma olhada aleatória: [edição 50](https://prefeituradecameta.pa.gov.br/wp-content/uploads/2023/04/Diario-Oficial-Eletronico-050-2023.pdf)